### DISCUSSION TITLE
How to `loop.turn()` and `loop.poll()` manually?

### DISCUSSION THREAD

**Discussion by BorMor** [capnproto/capnproto#1979] [Q&A]
Hello, I trying to use RPC with game engine, but I don't understand how to run event loop without blocking thread. All client examples end up with `.wait(waitScope)` but I want to use `.detach()` and then call `turn`/`poll` manually every game frame to execute promises.
Is it possible to achieve with `EzRpcClient` or I have to use more complex construction based on direct usage of `EventLoop` ?

**Comment by kentonv** [comment]
Have you tried `waitScope.poll()`?
